src/controllers/item.js

```diff
@@ -11,13 +18,41 @@ module.exports = {
   getFilteredItems: (req, res) => {
     res.send('Filtered items');
   },
-  updateItem: (req, res) => {
-    res.send('Item with the specified ID has been updated');
+  updateItem: async (req, res) => {
+    const { id } = req.params;
+    try {
+      const updatedItem = await ItemModel.findByIdAndUpdate(
+        id,
+        { $set: req.body },
+        { new: true }
+      );
+      if (!updatedItem) {
+        res
+          .status(422)
+          .json({ message: 'The item with the specified ID was not found.' });
```

Comment:
Excellent job checking for the existence of the item 👏 
You can throw an error here instead of sending the response as suggested in the previous PR 🙈